### ISSUE TITLE
error

### ISSUE THREAD

**Issue by rohitdesigner** [mongodb/mongo-php-library#309]
I WANT TO COONECT PHP7 WITH MONGODB .pls help me ...

**Comment by Servuc** [comment]
Hi (it's not useless ...)

Just install MongoDB (under Debian):

    pecl install mongodb-1.1.8
    echo extension=mongo.so > /usr/local/etc/php/conf.d/mongo.ini

Restart your web server to take care of change.

And in your code : 

     $this->m_mongo = new \MongoDB\Client("mongodb://" . $user . ":" . $password . "@mongodb:27017/" . $database);

**Comment by doublecompile** [comment]
@rohitdesigner What kind of error is occurring? Can you please share an exception stack trace?

**Comment by rohitdesigner** [comment]
yes pls... iam using php7 with mongodb but i am getting an error when deploying it on mamp..

![capture](https://cloud.githubusercontent.com/assets/17812216/21018259/8e3261ca-bcff-11e6-8270-a5c6a4e82555.PNG)
![hello](https://cloud.githubusercontent.com/assets/17812216/21018260/8e3521bc-bcff-11e6-9768-18b9637a1dd7.PNG)

# now this is the code
`<?php
require_once("/vendor/autoload.php");


   $database = (new MongoDB\Client)->example;
?>
  
**Note: i have installed mongodb php library from composer.**

when deploing :
![k](https://cloud.githubusercontent.com/assets/17812216/21018466/482b5596-bd00-11e6-8f09-46520e310435.PNG)


please help me... i am googling for the error since 15 days
 




`

**Comment by petr-buchin** [comment]
I don't know why your site does not work, but I see problem in this line:
```php
require_once("/vendor/autoload.php");
```

Leading slash in file path means that this is an absolute path. Probably, you should change this to 
```php
require_once("vendor/autoload.php");
```

But I think the problem is also in your environment, check it first.

**Comment by moderndeveloperllc** [comment]
@rohitdesigner Check https://github.com/mongodb/mongo-php-driver/issues/476. Others are having issues with Windows and the 1.2.0 driver.

**Comment by rohitdesigner** [comment]
can u connect through my pc

**Comment by rohitdesigner** [comment]
can u pls connect through teamveiwer to my pc and pls pls solve this

**Comment by petr-buchin** [comment]
@rohitdesigner , this is the Github page of the concrete project. Here you should post issues with this concrete library. 
But your problem is that you cannot get your environment up and running. 
This is not the issue you can post here.

**Comment by rohitdesigner** [comment]
environment is correct i think... you can see in the screenshot

**Comment by petr-buchin** [comment]
@rohitdesigner you should provide concrete issues - what error is occuring (for example, it can be logged in php-fpm log if you are using php-fpm). But you just posted screenshot, which can tell us, that your site is not working. 
If you want to get help - ask concrete question.

**Comment by petr-buchin** [comment]
@rohitdesigner , please read documentation of your web server and find, where php errors are logged in your environment. 
When you'll find this, you just should post your error text and stack trace here, and you will get help.

**Comment by jmikola** [comment]
Cross-referencing with:

 * https://github.com/mongodb/mongo-php-driver/issues/428 
 * https://github.com/mongodb/mongo-php-driver/issues/476 (as @moderndeveloperllc linked above)

If there is no PHP error and the web server log reports a crash, this is like the same issue. I am in the process of trying to reproduce the crash in the above issues in a Windows VM and will update those issues when I have something to report.

**Comment by rohitdesigner** [comment]
no log . its empty

**Comment by jmikola** [comment]
Please see my comment in https://github.com/mongodb/mongo-php-driver/issues/476#issuecomment-266022770. I believe we've diagnosed the root cause and I'm in the process of sorting it out.

**Comment by jmikola** [comment]
Closing this as a duplicate. I've also uploaded a DLL in https://github.com/mongodb/mongo-php-driver/issues/476#issuecomment-266088977 that may be used until we properly release the fix in a 1.2.2 release. If you have further comments, please direct them to the that thread.

**Comment by jmikola** [comment]
Just to follow up: [1.2.2](https://github.com/mongodb/mongo-php-driver/releases/tag/1.2.2) has been released with the relevant fix. Windows builds should show up on [PECL](pecl.php.net/package/mongodb) within a couple of hours.
